From 8a3fdaa201a599d29475456d24bdf4a541e6a218 Mon Sep 17 00:00:00 2001
From: Omair Majid <omajid@redhat.com>
Date: Wed, 23 Oct 2019 15:43:57 -0400
Subject: [PATCH 11/14] Support global.json on arm64 as well

arcade uses the runtime section of global.json to decide which
architecture + runtime combination needs to be installed.

With https://github.com/dotnet/arcade/pull/4132 arcade can install
foreign SDKs in separate locations correctly.

This change, suggested by @dougbu, makes arcade always install the
runtime for the local architecture (which means it should work on arm64
and x64) as well as the x86 architecture (skipped on Linux).

This gets us a working SDK/Runtime combo on arm64.
---
 global.json | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/global.json b/global.json
index b74e0f1129..83d7666c92 100644
--- a/global.json
+++ b/global.json
@@ -5,10 +5,7 @@
   "tools": {
     "dotnet": "3.1.117",
     "runtimes": {
-      "dotnet/x64": [
-        "$(MicrosoftNETCoreAppInternalPackageVersion)"
-      ],
-      "dotnet/x86": [
+      "dotnet": [
         "$(MicrosoftNETCoreAppInternalPackageVersion)"
       ]
     },
-- 
2.18.0

